openvalidators/reward_config.yaml

```diff
@@ -0,0 +1,11 @@
+# The 'reward_models_weights' configuration sets the distribution of weights used in the reward framework.
+# These weights determine the relative importance of each reward model.
+# It is important to note that all the weights must sum up to one (1) to ensure a proper balance in the reward calculations.
+#
+# Note: Fields should match RewardModelType Enum in openvalidators/config.py
+reward_models_weights:
```

Comment:
rlhf supercedes dahoas (so turn this off), and also outperforms prompt-based (so turn this off).
So the following weights may be safer for now:
```python
reward_models_weights:
   rlhf_reward_model: 0.4
   reciprocate_reward_model: 0.3
   dahoas_reward_model: 0
   diversity_reward_model: 0.3
   prompt_reward_model: 0
```